src/context/UserProvider.tsx

```diff
@@ -1,13 +1,11 @@
 import React, { createContext, FC, useContext, useMemo, useState } from 'react';
-import { useWallet, UseWalletProvider } from 'use-wallet';
 import useIdle from 'react-use/lib/useIdle';
 
-import { CHAIN_ID } from '../utils/constants';
-import { AVAILABLE_CONNECTORS } from '../utils/connectors';
+import { OnboardProvider, useWalletAddress } from './OnboardProvider';
 
 interface State {
-  account?: string;
-  masqueradedAccount?: string;
+  address: string | undefined;
+  masqueradedAccount: string | undefined;
```

Comment:
FYI these are functionally identical; best to use the `?` style for brevity imo.